datafusion/functions/src/math/log.rs

```diff
@@ -37,6 +38,7 @@ use datafusion_expr::{
 };
 use datafusion_expr::{ScalarUDFImpl, Signature, Volatility};
 
+#[udf_doc(description = "log_description", example = "log_example")]
```

Comment:
here we put the documentation

Reply:
This looks quite cool ❤️ 

One thing I would be interested in seeing is if/how a multi line description and example would look. I have had trouble with multi-line macro / docs in the past